ale_linters/ruby/rubocop.vim

```diff
@@ -25,20 +25,37 @@ function! ale_linters#ruby#rubocop#Handle(buffer, lines) abort
 endfunction
 
 function! ale_linters#ruby#rubocop#GetCommand(buffer) abort
-    return 'rubocop --format emacs --force-exclusion '
+    let l:unescaped = ale#Var(a:buffer, 'ruby_rubocop_executable')
+    let l:executable = ale#Escape(l:unescaped)
+    if l:unescaped =~? 'bundle$'
+        let l:executable = l:executable . ' exec rubocop'
+    endif
+    return l:executable
+    \   . ' --format emacs --force-exclusion '
     \   . ale#Var(a:buffer, 'ruby_rubocop_options')
     \   . ' --stdin ' . bufname(a:buffer)
 endfunction
 
+function! ale_linters#ruby#rubocop#GetExecutable(buffer) abort
+    let l:executable = ale#Var(a:buffer, 'ruby_rubocop_executable')
```

Comment:
You just need to `return` here, as ALE uses `if executable(...)` internally. I'll fix this later.